ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -90,6 +91,13 @@ Vivek Kini, NVIDIA +
 typedef cl_uint cl_external_semaphore_handle_type_khr;
 ----
 
+The `cl_khr_external_semaphore_sync_fd` extension adds:
+
+[source]
+----
+typedef cl_properties cl_semaphore_import_properties_khr;
```

Comment:
Should this be renamed to cl_semaphore_reimport_properties_khr?
Are these going to be distinct from cl_semaphore_properties_khr that can be passed at the time of clCreateSeamphoreKHR possibly supporting initial import?

Reply:
Can be merged after rename to cl_semaphore_reimport_properties_khr, version update and conflict resolution.